consensus/clique/clique.go

```diff
@@ -375,6 +380,14 @@ func (c *Clique) snapshot(chain consensus.ChainHeaderReader, number uint64, hash
 		snap    *Snapshot
 	)
 	for snap == nil {
+		if c.config.ShadowForkHeight > 0 && number == c.config.ShadowForkHeight {
```

Comment:
should we `number >= c.config.ShadowForkHeight`?

Reply:
Not needed, once we switch over to the new signer Clique will maintain it as the only signer.